sdk/lib/atl/atlbase.h

```diff
@@ -662,6 +665,127 @@ class CAtlDllModuleT : public CAtlModuleT<T>
     }
 };
 
+
+template <class T>
+class CAtlExeModuleT : public CAtlModuleT<T>
```

Comment:
Public declaration has an attribute ATL_NO_VTABLE (which is declared somewhere in our code). I don't know whether it is meaningful to add it there too.

Reply:
We have that on no class yet, but it should be on a ton.
Didn't want to break style